web/public/js/services/dojo-utils.js

```diff
@@ -135,24 +135,56 @@ angular.module('cpZenPlatform').factory('dojoUtils', ['$location', '$translate',
     { id: 'search_engine',
       name: $translate.instant('Search Engine')},
     { id: 'volunteers',
-      name: $translate.instant('Other CoderDojo Volunteers')},
-    { id: 'organisations',
-      name: $translate.instant('Other Coding Organisations')},
+      name: $translate.instant('CoderDojo Volunteers')},
+    { id: 'foundation',
+      name: $translate.instant('Raspberry Pi Foundation / CoderDojo Foundation outreach')},
+    { id: 'social',
+      name: $translate.instant('Social media')},
     { id: 'future_learn',
       name: 'Future Learn'},
-    { id: 'developpers',
-      name: $translate.instant('Development Community')},
+    { id: 'employer',
+      name: $translate.instant('My employer (CSR scheme at work)')},
```

Comment:
For any new string, or modified string that is translated, you need to modify https://github.com/CoderDojo/cp-translations/blob/master/strings/en_US/messages.po

Reply:
Ah ok 👍

Reply:
Do I need to update these as well?
https://github.com/CoderDojo/cp-zen-platform/blob/master/web/public/js/services/translation-keys.js#L158

Reply:
The idea behind the translationKeys was to not tie the text of the key to the frontend, so we could change the translation when necessary without touching the frontend.
We didn't use them much in the end (and changed frontend framework...)
So nope, don't worry about it